### PR TITLE
feat(language-list): apply new specs - INNO-1186

### PR DESCRIPTION
**Pull request by emeryro**
# PR description

Apply [component specifications](https://webgate.ec.europa.eu/CITnet/confluence/x/QBX-MQ)

Notes
- this is a template, not a component
- only EU version is provided (EC will be done in INNO-1180)

### CODE REVIEW

**Comment by HAL-Patch-INNO** [comment]
Deploy preview for *europa-component-library* ready!

Built with commit 8aa528fed3b7370ad8fbcf1ac1af90cc3f03a624

https://deploy-preview-817--europa-component-library.netlify.com

File: src/systems/ec/implementations/vanilla/packages/ec-component-language-list/ec-component-language-list.scss [hunk 1]
```diff
@@ -1,11 +1,10 @@
 /**
  * Language list component
- * @define language-list
+ * @define language-list; weak
```
Comment (yhuard) [inline src/systems/ec/implementations/vanilla/packages/ec-component-language-list/ec-component-language-list.scss]: 😞

File: src/systems/eu/implementations/react/packages/language-list/LanguageList.jsx [hunk 2]
```diff
@@ -0,0 +1,58 @@
+/* eslint-disable import/no-extraneous-dependencies */
```
Comment (yhuard) [inline src/systems/eu/implementations/react/packages/language-list/LanguageList.jsx]: Please remove this comment

File: src/systems/eu/implementations/react/packages/language-list/LanguageList.jsx [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+  return (
+    <div className="ecl-row">
+      <div className="ecl-language-list__column ecl-col-12 ecl-col-sm-4 ecl-offset-sm-2">
+        <ul className="ecl-language-list__list">
+          {column1.map(item => (
+            <li
+              className={`ecl-language-list__item ${
+                item.isActive ? 'ecl-language-list__item--is-active' : ''
+              }`}
```
Comment (yhuard) [inline src/systems/eu/implementations/react/packages/language-list/LanguageList.jsx]: I guess React is complaining about a missing `key`, isn't it?

File: src/systems/eu/implementations/react/packages/language-list/LanguageListItem.jsx [hunk 4]
```diff
@@ -0,0 +1,31 @@
+/* eslint-disable import/no-extraneous-dependencies */
```
Comment (yhuard) [inline src/systems/eu/implementations/react/packages/language-list/LanguageListItem.jsx]: Please remove comment

**Review by yhuard — DISMISSED** [review]
while waiting for netlify...

File: src/systems/eu/implementations/react/packages/language-list/LanguageList.jsx (hunk 3 above)
Comment (emeryro) [reply src/systems/eu/implementations/react/packages/language-list/LanguageList.jsx]: No, it isn't

Comment (yhuard) [reply src/systems/eu/implementations/react/packages/language-list/LanguageList.jsx]: yes, it is ;)

image

**Comment by yhuard** [comment]
image

is it the expected behaviour?

File: src/systems/eu/implementations/react/packages/language-list/LanguageList.jsx [hunk 5]
```diff
@@ -27,6 +28,7 @@ const LanguageList = ({ items }) => {
         <ul className="ecl-language-list__list">
           {column2.map(item => (
             <li
+              key={item.id}
```
Comment (yhuard) [inline src/systems/eu/implementations/react/packages/language-list/LanguageList.jsx]: the `item.label` could have been the key too ;)

Comment (emeryro) [reply src/systems/eu/implementations/react/packages/language-list/LanguageList.jsx]: indeed...

**Comment by yhuard** [comment]
image

This extra spacing should be reported to the DT or adjusted.

I don't see `ecl-container`anywhere while it should be used

**Review by degliwe — APPROVED** [review]

**Pull request merged**